yoyodyne/models/base.py

```diff
@@ -336,6 +337,18 @@ def _get_lr_scheduler(
             "interval": "step",
             "frequency": 1,
         }
+        # For reduce on plateau min, we reduce when val loss stops decreasing.
```

Comment:
This is really unexpected for me---is this standard practice to have both? If so why not call the two modes something like `--reduceonplateau_mode loss` and `--reduceonplateau_mode accuracy` instead?
If you want to stick with this, maybe you could spell this out in slightly less telegraphic language? I think something like:

```
# When using `reduceonplateau_mode min`, we reduce when validation loss stops decreasing. When using `reduceonplateau_mode max`, we reduce when validation accuracy stops increasing.
```

Reply:
The scheduler pretty much just follows torch: https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.ReduceLROnPlateau.html. I assume that `max` mode would only make sense for accuracy. However, I would expect `min` on validation loss to be the norm.

I can either force it to not expose the other options, or update the comment. Do you have a preference?

Reply:
I jsut think _calling them_ max and min is burying the lede, when what you're doing is "tracking plateauing loss" vs. "tracking plateauing accuracy". The underlying behavior from PyTorch doesn't really apply in the sense that you can pass any metric you want to it but we're just supporting these two. (It'd be crazy to reduce on min accuracy or max loss so let's perish the thought.)

Reply:
Ah I see. In that case I like your suggestions for `loss` or `accuracy` better, but I am also slightly hesitant to change too much from the pytorch interface. Right now I lean changing it though.

Reply:
There is an alternative (though I don't recommend it) in which you have both flags for the choice of metric and min vs. max. I don't think this is a good idea because I don't think you should expose crazy things to the user

  don't think min entails "validation loss" or max entails "validation accuracy" so it certainly should be more explicit than the current form.

Reply:
Does it look ok following just the first suggestion?